src/raster/r.in.vect/r.in.vect.py

```diff
@@ -60,6 +60,12 @@
 # % description: Set this flag if the whole vector layer needs to be converted. By default, only the part overlapping with the computational region is converted.
 # %end
 
+# %flag
+# % key: g
+# % label: Match region's extent to vector bounding box
+# % description: Set region extent to match that of the bounding box of the vector layer.
```

Comment:
What about using instead `-a` (as done in `g.region`) to "adjust" things? Since `-g` is commonly used for script-parsable output...

Reply:
Good point, changed.